src/api/firebase.js

```diff
@@ -165,12 +165,13 @@ export async function addItem(listPath, { itemName, daysUntilNextPurchase }) {
 	const listCollectionRef = collection(db, listPath, 'items');
 	// TODO: Replace this call to console.log with the appropriate
 	// Firebase function, so this information is sent to your database!
-	return console.log(listCollectionRef, {
+	const newItemEntry = doc(listCollectionRef);
+	return setDoc(newItemEntry, {
```

Comment:
because we are not trying to update items here, it would be better to use the `addDoc` function instead of `setDoc`. (it takes the listCollectionRef directly, you won't need `const newItemEntry = doc(listCollectionRef);`)

Also, you can remove these comments as you already replaced console.log